codeface_extraction/extractions.py

```diff
@@ -30,19 +33,22 @@
 #
 
 
-def get_extractions(dbm, conf, resdir, csv_writer):
+def get_extractions(dbm, conf, resdir, csv_writer, exclude_impl):
     # all extractions are sublcasses of Extraction:
     # instantiate them all!
     __extractions = []
     for cls in Extraction.__subclasses__():
-        __extractions.append(cls(dbm, conf, resdir, csv_writer))
+        if (exclude_impl or
```

Comment:
`!exclude_impl`, isn't it? This is likely a follow-up thing from the potential mistake with the new parser argument.